### ISSUE TITLE
JAWS stops tracking focus within webview after navigation

### ISSUE THREAD

**Issue by BrendanMcK** [electron/electron#13244]
* Electron Version: 2.0.0
* Operating System (Platform and Version): Windows 10
* Last known working Electron version: unknown

**Expected Behavior**

When a webview loads new content - eg navigating to another page, or reloading current one, focus may get reset to the start of the page, but it should continue working as normal from there. As the user hits TAB, JAWS should move to the next focusable item, and announce its name and role. This is what happens in regular Chrome browser.

**Actual behavior**

When a WebView refreshes, JAWS seems to lose track of where the focus is. As you tab between the items in the refreshed webview, JAWS announces 'tab' as you press tab to navigate, but does not announce the name or role of the element that now has focus. If the element is an input, JAWS remains in 'web' mode - so if you press H (for example), instead of the h being typed into the input field, JAWS interprets it as a command to move to the next Heading.

It's possible to force JAWS back into a good state by clicking away to another app then clicking back to the web app, but this isn't a viable workaround for a JAWS users.

**To Reproduce**

Repro based on electron-quick-start available here: https://github.com/BrendanMcK/electron-quick-start/tree/bmckeon-jaws-focus-lost

If you provide a URL, please list the commands required to clone/setup/run your repo e.g.
```sh
$ git clone https://github.com/BrendanMcK/electron-quick-start.git -b bmckeon-jaws-focus-lost
$ npm install
$ npm start
```

When the page initially loads, with JAWS running, you can tab / shift-tab between the username and password fields, and JAWS reads out the correct information. After hitting Enter to submit the form and trigger a reload, JAWS only reads out 'tab' as you do this.

**Additional Information**

It appears to be possible to work-around this by:
* Adding a dummy `div` to the index.hml, alongside the webview, with tabindex=-1
* When the webview is refreshed - detect by listening for dom-ready event - set focus to the dummy div, then set focus to the webview.
* If code within the webview does any focus maintenance as part of its init code, it may need to defer so that it does so after the webview changes the focus - otherwise focus may remain on the webview itself rather than the target child element.

Example workaround here: https://github.com/BrendanMcK/electron-quick-start/tree/bmckeon-jaws-focus-workaround

Ideally Electron should be sending the necessary events so that using a WebView component *just works* without the need for a workaround that may not even realize is needed.

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v4.2.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
